content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,276 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Application Terms::
+(To be defined)
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+* Source: 
+* Notes: Taken over from terminology collection list on Sharepoint.
+
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
+
+* Source: 
+* Notes: 
+
+3D Environment::
+(To be defined)
```

Comment:
Proposed definition: 3D representation of the surrounding of a simulated sensor. It 3D environment can consist of multiple 3D models.

Question? Should be include the material parameters in this definition? In that case I would state that in the definition and change "3D models" to "3D assets"